test/native/integration-test-helpers/add-block.js

```diff
@@ -11,13 +11,16 @@ import { waitFor } from './wait-for';
 /**
  * Adds a block via the block picker.
  *
- * @param {import('@testing-library/react-native').RenderAPI} screen    A Testing Library screen.
- * @param {string}                                            blockName Name of the block to be inserted as shown in the block picker.
+ * @param {import('@testing-library/react-native').RenderAPI} screen                 A Testing Library screen.
+ * @param {string}                                            blockName              Name of the block to be inserted as shown in the block picker.
+ * @param {boolean}                                           [isPickerOpened=false] isPickerOpened Option to skip opening the inserter picker.
  */
-export const addBlock = async ( screen, blockName ) => {
+export const addBlock = async ( screen, blockName, isPickerOpened = false ) => {
```

Comment:
I saw that for options we usually grouped them in a single object ([reference](https://github.com/WordPress/gutenberg/blob/afce9042bd0260aa8350d9412da3e027c23d58a2/test/native/integration-test-helpers/get-block.js#L10)), this way we made the entire object to be optional and options are named arguments, wdyt?
```suggestion
export const addBlock = async (
	screen,
	blockName,
	{ isPickerOpened } = {}
) => {
```

Reply:
Good idea! I'll update it, thanks!

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/46822/commits/bc09c4c1fcd630074d7985335abec59ecae944f0